src/core_plugins/kibana/public/management/styles/main.less

```diff
@@ -12,10 +12,6 @@ kbn-management-objects-view {
   display: block;
 }
 
-nav.navbar {
-  min-height: 70px;
-}
-
 .tab-management {
   background-color: @kibanaGray6;
 }
```

Comment:
Could you instead create a class here that's specific to the management navbar?

```less
/**
 * 1. Allow navbar to get taller on narrow screens.
*/
.management-navbar {
  min-height: 70px; /* 1 */
}
```

And then apply this class in `core_plugins/kibana/public/management/app.html`? This will help us keep the style scoped to a particular part of the UI, improving maintainability.